packages/subiquity_client/lib/subiquity_client.dart

```diff
@@ -24,11 +24,27 @@ class SubiquityClient {
     return _client.flush().then((_) => _client.close());
   }
 
-  Future<void> checkStatus(String method, StreamedResponse response) async {
+  String _formatResponse(String method, String response) {
+    var formatted = response;
+    if (response.length > 120) {
+      formatted = '${response.substring(0, 120)}...';
```

Comment:
You might want to make `120` a constant, to ease changing that value if we decide to in the future.